src/blocks/HeroImageBlockBrandPivot/HeroImageBlockBrandPivot.stories.tsx

```diff
@@ -0,0 +1,85 @@
+import { boolean, select, withKnobs } from '@storybook/addon-knobs'
+import * as React from 'react'
+import { MarkdownHtmlComponent } from '../BaseBlockProps'
+import {
+  HeroImageBlockBrandPivot,
+  HeroImageBlockBrandPivotProps,
+} from './HeroImageBlockBrandPivot'
+
+export default {
+  title: 'Blocks/HeroBlock',
+  component: HeroImageBlockBrandPivot,
+  decorators: [withKnobs],
+}
+
+const heroText: MarkdownHtmlComponent = {
+  _uid: '1234',
+  html:
+    '<p>Hedvig är framtidens försäkring. Medan traditionella försäkringsbolag försäkrar prylar, hjälper vi människor. Vi finns specifikt för dig i behov av omedelbar hjälp, oavsett var du befinner dig.</p>',
+  original:
+    '<p>Hedvig är framtidens försäkring. Medan traditionella försäkringsbolag försäkrar prylar, hjälper vi människor. Vi finns specifikt för dig i behov av omedelbar hjälp, oavsett var du befinner dig.</p>',
+  plugin: 'markdown-html',
+}
+
+const colors = {
+  standard: {
+    _uid: '6ecde11d-ba0a-48fb-9b7b-e6dbf31415d9',
+    color: 'standard',
+    plugin: 'hedvig_minimal_color_picker',
+  },
+  standardInverse: {
+    _uid: '6ecde11d-ba0a-48fb-9b7b-e6dbf31415d9',
+    color: 'standard-inverse',
+    plugin: 'hedvig_minimal_color_picker',
+  },
+  gray700: {
+    _uid: '6ecde11d-ba0a-48fb-9b7b-e6dbf31415d9',
+    color: 'gray700',
+    plugin: 'hedvig_minimal_color_picker',
+  },
+  gray500Inverse: {
+    _uid: '6ecde11d-ba0a-48fb-9b7b-e6dbf31415d9',
+    color: 'gray500-inverse',
+    plugin: 'hedvig_minimal_color_picker',
+  },
```

Comment:
these'll be used in more places no? should we extract it to somewhere where its reusable?

Reply:
that might be good! what do you think about some utils for storybook?

Reply:
I've created a `utils/storybook.ts` and moved it there.